### ISSUE TITLE
May you please add an option so the client won't open bot intents in channels, groups or private chats?

### ISSUE THREAD

**Issue by 0xRustlang** [forkgram/TelegramAndroid#127]
Hello

Some channels have some bot intents like this:

https://t.me/your_bot?start=airplane

(More info in deeplinking section here: https://core.telegram.org/bots/features#deep-linking)

Some channels use this method to spread pornographic content.

Those channels are alright but unfortunately they get advertising and get money this way.

My son joined several channels that somehow are usefull but do this method and my son always intentionally or mistakenly start those bot and watch pornographic videos.

May you please add an option so that we can disable these?

May you please add an option so the client won't open bot intents in channels, groups or private chats?

I mean when user set that option, then forkgram won't process those intents.

I think checking the word "bot?start" in the URLs that user taps on them can be enough and it needs to just check URLs that have these domains in them:

t.me
telegram.me
telegram.dog

Thank you very much

**Comment by 23rd** [comment]
Please check a new option in the latest 9.4.0.0. version.

**Comment by 0xRustlang** [comment]
> Please check a new option in the latest 9.4.0.0. version.

Thank you so much.
As I have installed the F-droid version for my son because of easier updates, I couldn't check that but I checked the added names in latest commit and I want to thank you.

Probably that option will eliminate at least 80 percent of those annoying adult bots plus more privacy for some users.

Thank you very much for implementation of it, your already great fork, kind helps and replies and being open to suggestions.

I am really grateful for your efforts. 
Thanks you.
Best wishes.
